staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -958,6 +965,12 @@ func DefaultBuildHandlerChain(apiHandler http.Handler, c *Config) http.Handler {
 	handler = genericapifilters.WithAudit(handler, c.AuditBackend, c.AuditPolicyRuleEvaluator, c.LongRunningFunc)
 	handler = filterlatency.TrackStarted(handler, c.TracerProvider, "audit")
 
+	// REMOVE once testing in CI is done
+	if c.AnnotateEarlyAndLateRequests || true {
+		handler = genericfilters.WithShutdownLateAnnotation(handler, c.lifecycleSignals.ShutdownInitiated, c.ShutdownDelayDuration)
```

Comment:
if you're breaking down this way, determining the various stages of a shutdown
1. normal running
2. term received, readyz=false, normal handling
3. reject new connections
4. close old connections

would be natural.

Reply:
the way handler chain is setup today, after shutdown-delay-duration elapses, requests are not saved in audit. Let's assume this is not true, this PR starts adding annotation since TERM received and continues as long long as it receives request. 

If we see audit logs that shows threshold > 100%  then we can add more details